README.md

```diff
@@ -492,6 +493,33 @@ Team Exiv2 have not explored this configuration and cannot provide support.
 
 I've never succeeded in getting this to work.  I use different VMs for Linux 32 and 64 bit.  I've documented how to set up Cygwin and MinGW/msys2 for 64 and 32 bit builds.
 
+[TOC](#TOC)
+<name id="2-13">
+### 2.13 Building Exiv2 with ccache
+
+To speed up compilation, the utility ccache can be installed to cache the output of the compiler.  This greatly speeds up the build when you frequently build code that has not been changed.
+
+Installing and using ccache (and other similar utilities), is platform dependent.  On Ubuntu:
+
+```bash
+$ sudo apt install --yes ccache
+```
+
+To build with ccache, use the cmake option **-DBUILD\_WITH\_CCACHE=On**
+
+```bash
+$ cd <exiv2dir>
+$ mkdir build ; cd build ; cd build
+$ cmake .. -G "Unix Makefiles" -DBUILD\_WITH\_CCACHE=On
+$ make
+# Build again to appreciate the performance gain
+$ make clean
+$ make
+```
+
+Due to the way in which ccache is installed in Fedora, this option is defeated on that platform.  [https://github.com/Exiv2/exiv2/issues/361](https://github.com/Exiv2/exiv2/issues/361)
```

Comment:
This sounds like ccache does not work on Fedora, which is no true. This option just can't turn it off, that's all.

Reply:
Ah @D4N Thanks for the clarification.  As I said in #361 "This is very confusing.".  I'll update the documentation tomorrow.

Reply:
@piponazo I've changed the code in cmake/packaging.cmake that detects clang.